bindings/rust/s2n-tls-tokio/tests/common/time.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use s2n_tls::callbacks::ClockCallback;
+use std::time::Duration;
+use tokio::time::Instant;
+
+/// A monotonic clock that allows the s2n-tls C library time
+/// to follow the tokio::time::pause behavior.
+pub struct TokioTime(Instant);
```

Comment:
It might be a good idea to wire this clock up in the tokio bindings by default

Reply:
It's useful in tests because tokio lets you mess with time for tests, but is it useful in production? The default monotonic clock should be fine as long as tokio isn't pausing / fast-forwarding time.

Reply:
Well I think applications also like to test so wiring the timing up by default to the tokio clock will make that easier. But you're right, in production it's rarely used.

Reply:
Looking into making this change, the clock callback is on the config instead of the connection, so because configs are created and provided by the caller we can't easily set a default. We'd either need to add a method to get a default config for use with these bindings, or add a connection-level version of the callback, which seems like overkill.

Reply:
Makes sense